app/serializers/echonest.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-data-echonest/serializers/echonest';
```

Comment:
`No newline at the end of file` warning? Some people care and some don't. Alex cared a lot so just wondering.